tests/core/test_schema_loader.py

```diff
@@ -129,7 +133,7 @@ def test_no_internal_model_conversion(tmp_path: Path, mocker: MockerFixture):
     assert len(schema) == 2
     assert schema[0]["name"] == "`tbl-d`"
     assert list(schema[0]["columns"]) == ["b", "a"]
-    assert schema[1]["name"] == "tbl_c"
+    assert schema[1]["name"] == "`tbl_c`"
```

Comment:
@tobymao This name changed to now have backticks. Is this ok? Not sure the intent behind this test.

Reply:
yes, seems ok